### PR TITLE
[c-sample] First version. Fixes JB#58945

### PR DESCRIPTION
**Pull request by vigejolla**

### CODE REVIEW

File: rpm/c-sample.spec [hunk 1]
```diff
@@ -0,0 +1,29 @@
+Name:       c-sample
+Summary:    Sample app written in C
+Version:    0
+Release:    1
+License:    BSD
+URL:        https://www.github.com/sailfish/c-sample
+Source0:    %{name}-%{version}.tar.bz2
+Requires:   ssu-sysinfo
```
Comment (rainemak) [inline rpm/c-sample.spec]: Not needed. Comes via BuildRequires.

File: src/c-sample.c [hunk 2]
```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2022, Jolla Ltd.
```
Comment (rainemak) [inline src/c-sample.c:2]:
```suggestion
 * Copyright (c) 2022 Jolla Ltd.
```
Official syntax is without comma

Comment (vigejolla) [reply src/c-sample.c:2]: I literally copypasted this from the example from the official document, it has a comma

**Review by rainemak — APPROVED** [review]

File: rpm/c-sample.spec [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+%description
+Simple app written in C
+
+%prep
+%autosetup -p1 -n %{name}-%{version}
+
+%build
+cd src
+gcc -o c-sample c-sample.c -lssusysinfo
```
Comment (rainemak) [inline rpm/c-sample.spec:18]: I'd actually take both --clags and --libs from the ssu-sysinfo pkgconfig.

> gcc -o c-sample c-sample.c $(pkg-config --cflags --libs ssu-sysinfo)

Comment (vigejolla) [reply rpm/c-sample.spec:18]: > I'd actually take both --clags and --libs from the ssu-sysinfo pkgconfig.
> 
> > gcc -o c-sample c-sample.c $(pkg-config --cflags --libs ssu-sysinfo)

I'll make that part of the Makefile PR, if that's ok with you?

Comment (rainemak) [reply rpm/c-sample.spec:18]: Let this be an intentional mistake for few days :-)

**Pull request merged**